src/shogun/lib/any.h

```diff
@@ -259,18 +321,19 @@ namespace shogun
 
         BaseAnyPolicy* policy;
         void* storage;
+		EnumType enumtype;
     };
 
     inline bool operator==(const Any& lhs, const Any& rhs)
     {
         void* lhs_storage = lhs.storage;
-        void* rhs_storage = rhs.storage;        
+        void* rhs_storage = rhs.storage;
         return lhs.policy == rhs.policy and
             lhs.policy->equals(&lhs_storage, &rhs_storage);
     }
 
     inline bool operator!=(const Any& lhs, const Any& rhs)
-    {        
```

Comment:
you should start using a proper texteditor, one that doesnt override things in files